### PR TITLE
v1.9 backports 2021-02-02

### PR DESCRIPTION
**Pull request by qmonnet**
* #13989 -- runtime: specify ICMP ids on connectivity test (@kkourt)
 * #14709 -- node-neigh: Fix node removal and invalid neigh entry due to buggy arping response correlation (@brb)
    * *Skipped commit e0b41fe9c5b8* ("arping: Improve response correlation"):
      Arping library is not yet part of Cilium in v1.9.
 * #14721 -- datapath: remove SNAT maps entries when kube-proxy is enabled (@mazzy89)
 * #14818 -- Fix wrong url (@manuelbuil)
 * #14828 -- images/runtime: fix loopback CNI installation plugin (@aanm)

Once this PR is merged, you can update the PR labels via:
```upstream-prs
$ for pr in 13989 14709 14721 14818 14828; do contrib/backporting/set-labels.py $pr done 1.9; done
```

### CODE REVIEW

File: images/cilium/Dockerfile
Comment (pchaigno) [inline images/cilium/Dockerfile]: You might need to update this again once https://github.com/cilium/cilium/pull/14798 is merged.

Comment (qmonnet) [reply images/cilium/Dockerfile]: The GitHub action is blocking on that at the moment anyway :/. `scripts/update-cilium-runtime-image.sh` provides a different tag, but I haven't figured out yet if I should just use the tag it retrieves or if I'm missing an update somewhere. [EDIT: André told me to use the tag provided by the script, and to ask Ilya for the internal details. I just did the former, and will do the latter.]

(But thanks for the notice!)

Comment (qmonnet) [reply images/cilium/Dockerfile]: Ok, waiting for your PR to be merged to rebase and run the CI. I'll un-draft in the meantime, as it is ready for reviews by original committers.

**Review by aanm — APPROVED** [review]
look good for my commit

**Review by brb — APPROVED** [review]
Thanks, LGTM.

File: images/cilium/Dockerfile
Comment (pchaigno) [reply images/cilium/Dockerfile]: @qmonnet The other backport PR was merged, you can rebase now.

**Comment by qmonnet** [comment]
test-backport-1.9

**Comment by qmonnet** [comment]
Screenshot_20210203_123506

I'll do my best :relaxed:

**Comment by qmonnet** [comment]
retest-runtime

**Comment by qmonnet** [comment]
test-gke

**Review by kkourt — APPROVED** [review]
LGTM for my commit.

**Comment by joestringer** [comment]
https://jenkins.cilium.io/job/Cilium-PR-Ginkgo-Tests-K8s/3958/ is not marked as required as it's generally failing on this branch. No need to block the backport merge on that, merging this PR.

**Pull request merged**
